src/NoBookingPage/index.js

```diff
@@ -15,6 +15,7 @@ import routeDefinitions from '../routeDefinitions';
 
 const style = css`
   div.NoBookingPage {
+    max-width: 480px;
```

Comment:
Apparently this fixed the issue (I checked with mobile and desktop, logged in and logged out).

But if I put

```
width: 480px;
```

it breaks other stuff, for example the breadcrumbs :joy:  Somebody explain why that is....